### ISSUE TITLE
Error loading syntax file "Packages/Nunjucks Syntax/Nunjucks.tmLanguage": Wrong type for key begin expected string

### ISSUE THREAD

**Issue by jitendravyas** [robmoggach/sublime-nunjucks#6]
Mac Yosemite
Sublime Text 3

**Comment by tkane2000** [comment]
+1

**Comment by rmobis** [comment]
I have the same issue on my ST3, Build 3095 on Windws 10.

**Comment by Spheerys** [comment]
Same issue here with ST3 under Linux
Solved by saving this file : https://raw.githubusercontent.com/mogga/sublime-nunjucks/master/Nunjucks.tmLanguage
... inside the right directory/file : "Packages/Nunjucks Syntax/Nunjucks.tmLanguage"

**Comment by Hlanderd** [comment]
Yep, that worked on mac too, but you have to create the directory:
/Users/_YOUR_USERNAME_/Library/Application Support/Sublime Text 3/Packages/**Nunjucks Syntax**

thx **Spheerys**!

**Comment by flepp** [comment]
Tried it on windows 8, no luck. Swig syntax highlighting works, so I think I'll stick with it for now.

**Comment by JamesJosephFinn** [comment]
yeah, jinja2 syntax package works like a charm. this package may not even be needed.

**Comment by optimix-designer** [comment]
Yes, the solution above works perfect (Sublime Text 3, Mac OS).
Thanks a lot guys! :)

**Comment by connor11528** [comment]
Worked for me! 

```
cd Library/Application\ Support/Sublime\ Text\ 3/Packages
mkdir "Nunjucks Synatx"
touch Nunjucks\ Synatx/Nunjucks.tmLanguage
```

And then paste in file from here: https://raw.githubusercontent.com/mogga/sublime-nunjucks/master/Nunjucks.tmLanguage

When you go back to sublime, in the bottom right corner select the language to Nunjucks

**Comment by CrescentCraftorium** [comment]
Little late to the party, but this is how I got it working with Sublime Text 3 on Windows 7 (work) and Windows 10 (home) ...
Install Nunjucks through Package Control
Navigate to Preferences > Browse Packages...
Create a new folder named 'Nunjucks Syntax'
Paste the above Nunjucks.tmLanguage file into the new folder you just made.
